libbeat/docs/newbeat.asciidoc

```diff
@@ -279,30 +315,57 @@ type Config struct {
 var DefaultConfig = Config{
 	Period: 1 * time.Second,
 }
+
```

Comment:
@ruflin Just an FYI that I added the empty line to the code example to fix the preview in GitHub. Before I added the extra line, the preview was messed up. Instead of seeing the formatted doc, you saw the asciidoc markup after this line. The extra line looks OK in the built doc, so I'll just leave it there for now.